docs/examples/params/model.yaml

```diff
@@ -0,0 +1,317 @@
+# Type of model that the rest of these parameters apply to.
```

Comment:
Do you mean other places in the codebase where we have model params with detailed comments? If yes, I agree.

I think this file is a great place for it, because it will show up here:

http://docs.numenta.org/quick-start/example-model-params.html

Perhaps we should remove the comments from other places and refer them to the API docs?